chsdi/models/vector/dritte.py

```diff
@@ -63,15 +63,18 @@ class PronaturaWaldreservate(Base, Vector):
     __table_args__ = ({'schema': 'pronatura', 'autoload': False})
     __template__ = 'templates/htmlpopup/pronatura.mako'
     __bodId__ = 'ch.pronatura.waldreservate'
-    __queryable_attributes__ = ['name', 'objnummer', 'gesflaeche', 'gesflaeche', 'gisteilobjekt', 'mcpfe']
+    __queryable_attributes__ = ['name', 'objnummer', 'gesflaeche', 'gesflaeche', 'gisteilobjekt']
```

Comment:
you can remove the whole line from both layers ch.pronatura.waldreservate/ch.bafu.waldreservate, this feature has not been used at all in 2020 and 2021.